### DISCUSSION TITLE
I cannot hear myself on Jamulus

### DISCUSSION THREAD

**Discussion by Hughlevinson** [jamulussoftware/jamulus#2455] [Technical Support & Help]
Any help would be greatly appreciated. I’ve been using Jamulus on a Macbook Air running OS Catalina 10.15.7 for about a year. I use an Audient iD4 box to connect up a mic and a guitar. Suddenly, it’s not working. I can connect to a server and hear my bandmates playing, but I can’t hear myself, and they can’t hear me. I have gone through the usual things eg disabling the inbuilt mic for Jamulus in the privacy settings: and enabling the Audient in and outs in the audio settings. I know that the Mac is connecting via the Audient box, as I can hear my bandmates. I have also tried downloading the legacy version of Jamulus to try that. Despite reinstalling the Jamulus software a couple of times, I encounter the same problem, both with the current Mac OS version and the legacy version. Any suggestions for help would be gratefully received. Thanks!

**Comment by pljones** [comment]
If you replace Jamulus by other software, can you hear yourself _through_ the software?

**Reply by gilgongo** [reply]
> If you replace Jamulus by other software, can you hear yourself _through_ the software?

Using Audacity, for instance.

**Reply by dcorson-ticino-com** [reply]
Maybe a cable problem between your interface and the microphone?
You could try using the internal mic and headphones, do they work?
